src/Chainweb/Chainweb.hs

```diff
@@ -355,7 +355,7 @@ withChainwebInternal conf logger peer rocksDb inner = do
                 }
 
             -- update the cutdb mvar used by pact service with cutdb
-            void $! putMVar mv mCutDb
+            void $! putMVar cdbv mCutDb
```

Comment:
More descriptive variable name from a previous iteration where we were doing index lookup.